### DISCUSSION TITLE
Push Monitoring (i.e. HTTP Webhook) feature

### DISCUSSION THREAD

**Discussion by projx** [apache/hertzbeat#694] [Q&A]
Hi All

I suspect the answer is no, as I cannot find mention of it... But does Hertzbeat have a "Push" monitor, i.e. a URL that I can call from my shell scripts, so it Hertzbeat records/shows the state of the cron.

Thanks

**Comment by tomsun28** [comment]
hi the feature `push metrics` is on the roadmap, and will come soon

**Reply by rellyrell7898** [reply]
Don't merge any changes

**Reply by rellyrell7898** [reply]
# - [ ] ![Screenshot_20230412-203437.png](https://user-images.githubusercontent.com/129052668/231662409-3ebba75f-490b-4465-9140-b6396cc68b24.png)
